test/collection_test.js

```diff
@@ -570,17 +570,17 @@ describe("Collection", () => {
         .then(res => articles.get(res.data.id))
         .then(res => {
           return articles.update(
-            Object.assign({}, {id: res.data.id}, {
-              title: "new title",
-            }));
+            {id: res.data.id,
```

Comment:
updated.

On Thu, Jun 16, 2016 at 5:35 PM, Rémy HUBSCHER <notifications@github.com>
wrote:

> In test/collection_test.js
> <https://github.com/Kinto/kinto.js/pull/471#discussion_r67332230>:
>
> > @@ -570,17 +570,17 @@ describe("Collection", () => {
> >          .then(res => articles.get(res.data.id))
> >          .then(res => {
> >            return articles.update(
> > -            Object.assign({}, {id: res.data.id}, {
> > -              title: "new title",
> > -            }));
> > +            {id: res.data.id,
>
> return articles.update({id: res.data.id, title: "new title"})
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Kinto/kinto.js/pull/471/files/cbb2dcf322216ccc623f5aba8bc1a0b67e1bc61c#r67332230>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFNMQf3sv7ct4E-or4YeUXFcWlNxZChxks5qMTv0gaJpZM4I2P0m>
> .
>



-- 

*Lavish Aggarwal*  | Mozilla Reps
| Google Student Ambassador|
India